Spanner/src/Operation.php

```diff
@@ -553,6 +553,7 @@ public function createSnapshot(Session $session, array $res = [], $className = S
      *           labels can be associated with a given session. See
      *           https://goo.gl/xmQnxf for more information on and examples of
      *           labels.
+     *     @type string $creator_role The session owner database role.
```

Comment:
pls check whether it should be `public` or `null`, if its `public` then we can add following:

```suggestion
     *     @type string $creator_role The session owner database role.
                 **Defaults to** `public`.
```

Reply:
Added **Defaults to** `null`.

Reply:
So, no need to mention the default here I think, right?

Reply:
If its `null`, then we are good.